infra/src/main/java/com/redhat/service/smartevents/infra/models/dto/BridgeStatusWrapperDTO.java

```diff
@@ -0,0 +1,63 @@
+package com.redhat.service.smartevents.infra.models.dto;
+
+import java.util.Objects;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.redhat.service.smartevents.infra.exceptions.BridgeError;
+
+public class BridgeStatusWrapperDTO {
```

Comment:
We used to send just the `Bridge` to the `manager` from the `operator`.

I now return the `Bridge` and any `BridgeError` that led to the `operator` to fail.